usr/bin/whonix-gateway-firewall

```diff
@@ -225,6 +225,9 @@ variables_defaults() {
    [ -n "$FLASHPROXY_PORT" ] || FLASHPROXY_PORT="9000"
 
    if [ ! "$WORKSTATION_ALLOW_SOCKSIFIED" = "1" ]; then
+      if [ "${info_enabled}" = "1" ]; then
```

Comment:
Perhaps drop this `if`? If a user changes such a setting, it seems good to always point it out. If the user changed something that could later be confusing the the user or later during support, then it seems good to point it out in an obvious way.

> Worried user questions which will be caused by this if too verbose by default:

Just showing always for users who don't change any config option always the full list of ports would be too verbose, confusing, I think.

Reply:
https://github.com/Whonix/whonix-firewall/pull/1/commits/b303e37017a25fe6893cc739cde430755f3c1f92